operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -120,6 +120,10 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
```

Comment:
My only concern adding autoscaling row in this position is that we break a bit the "flow" of the dashboard. The original idea of reads/writes dashboards was to show 1 line per hop in the network path, so you can quickly look at gateway --> distributor --> ingester.

What if we move the autoscaling rows down the dashboards, showing gateway and distributor autoscaling in consecutive rows?

If you agree, same comment applies to Reads dashboard.

Reply:
Checking.

Reply:
@pracucci I've moved auto-scaling rows in Reads dashboard beneath the "Store-gateway" row, and the auto-scaling rows in Writes dashboard beneath the "Ingester" row. Is that OK?